app/assets/stylesheets/_buttons.scss

```diff
@@ -1,265 +1,279 @@
+  -webkit-transition: color 0.3s cubic-bezier(0.7,0,0.3,1);
+  transition: color 0.3s cubic-bezier(0.7,0,0.3,1);
 }
 
 @mixin cubic-bezier-transform() {
-	-webkit-transition: transform 0.3s cubic-bezier(0.7,0,0.3,1);
-	transition: transform 0.3s cubic-bezier(0.7,0,0.3,1);
+  -webkit-transition: transform 0.3s cubic-bezier(0.7,0,0.3,1);
+  transition: transform 0.3s cubic-bezier(0.7,0,0.3,1);
 }
 
 @mixin circle-transition() {
-    -webkit-transition: -webkit-transform 0.2s, opacity 0.2s;
-    transition: transform 0.2s, opacity 0.2s;
-    -webkit-transition-timing-function: cubic-bezier(0.7,0,0.3,1);
-    transition-timing-function: cubic-bezier(0.7,0,0.3,1);
-    -webkit-transform: scale3d(0,0,1);
-    transform: scale3d(0,0,1);
+  -webkit-transition: -webkit-transform 0.2s, opacity 0.2s;
+  transition: transform 0.2s, opacity 0.2s;
+  -webkit-transition-timing-function: cubic-bezier(0.7,0,0.3,1);
+  transition-timing-function: cubic-bezier(0.7,0,0.3,1);
+  -webkit-transform: scale3d(0,0,1);
+  transform: scale3d(0,0,1);
 }
 
 @mixin translate-vertical($pixels) {
-    -webkit-transform: translate3d(0, $pixels ,0);
-    transform: translate3d(0, $pixels ,0);
+  -webkit-transform: translate3d(0, $pixels ,0);
+  transform: translate3d(0, $pixels ,0);
 }
 
 
 .tabs-style-circle {
-    .tabs-row {
-        padding-top: 50px;
-        padding-bottom: 50px;
-    }
-	overflow: visible;
-    li::before {
-        position: absolute;
-        top: 50%;
-        left: 50%;
-        margin: -60px 0 0 -60px;
-        width: 120px;
-        height: 120px;
-        border: 1px solid $medium-green;
-        border-radius: 50%;
-        content: '';
-        opacity: 0;
-        @include circle-transition();
-    }
-    li {
-        a {
-            color: $charcoal;
-        }
-    }
-    li.tab-current {
-        a {
-            color: $medium-green;
-            span {
-                //@include translate-vertical(4px);
-            }
-        }
-        &::before {
-            opacity: 1;
-            -webkit-transform: scale3d(1,1,1);
-            transform: scale3d(1,1,1);
-        }
-        .icon::before {
-            @include translate-vertical(-4px);
-        }
+  .tabs-row {
+    padding-top: 50px;
+    padding-bottom: 50px;
+  }
+
+  overflow: visible;
+  li::before {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin: -60px 0 0 -60px;
+    width: 120px;
+    height: 120px;
+    border: 1px solid $medium-green;
+    border-radius: 50%;
+    content: '';
+    opacity: 0;
+    @include circle-transition();
+  }
+
+  li {
+    a {
+      color: $charcoal;
     }
+  }
+
+  li.tab-current {
     a {
-        overflow: visible;
-        color: $charcoal;
-        font-weight: 400;
-        font-size: 0.9em;
-        line-height: 1.1;
-        @include cubic-bezier-color();
-        span {
-            display: inline-block;
-            @include cubic-bezier-transform();
-        }
+      color: $medium-green;
+      span {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 4